seed/views/v3/facilities_plan_run.py

```diff
@@ -223,3 +223,276 @@ def run(self, request, pk):
+                "annotated_name": "ci",
+                "display_name": "CI",
+                "queryable_name": F("extra_data__CI"),
+            },
+            {
+                "annotated_name": "replacement_cost_col",
+                "display_name": "Replacement Cost",
+                "queryable_name": F("replacement_cost"),
+            },
+            {
+                "annotated_name": "capacity_cooling",
+                "display_name": "Capacity_cooling",
+                "queryable_name": F("extra_data__CAPACITY_COOLING"),
+            },
+            {
+                "annotated_name": "capacity_cooling_units",
+                "display_name": "Capacity_cooling_units",
+                "queryable_name": F("extra_data__CAPACITY_COOLING_UNITS"),
+            },
+            {
+                "annotated_name": "capacity_heating",
+                "display_name": "Capacity_heating",
+                "queryable_name": F("extra_data__CAPACITY_HEATING"),
+            },
+            {
+                "annotated_name": "capacity_heating_units",
+                "display_name": "Capacity_heating_units",
+                "queryable_name": F("extra_data__CAPACITY_HEATING_UNITS"),
+            },
+            {
+                "annotated_name": "flowrate",
+                "display_name": "Flowrate",
+                "queryable_name": F("extra_data__FLOWRATE"),
+            },
+            {
+                "annotated_name": "flowrate_units",
+                "display_name": "Flowrate_units",
+                "queryable_name": F("extra_data__FLOWRATE_UNITS"),
+            },
+            {
+                "annotated_name": "fuel_type",
+                "display_name": "Fuel_type",
+                "queryable_name": F("extra_data__FUEL_TYPE"),
+            },
+            {
+                "annotated_name": "power",
+                "display_name": "Power",
+                "queryable_name": F("extra_data__POWER"),
+            },
+            {
+                "annotated_name": "power_units",
+                "display_name": "Power_units",
+                "queryable_name": F("extra_data__POWER_UNITS"),
+            },
+        ]
+        annotations = {cd["annotated_name"]: cd["queryable_name"] for cd in element_columns_dict}
+        elements = elements.annotate(**annotations)
+        elements_records = list(elements.values("property_id", *[cd["annotated_name"] for cd in element_columns_dict]))
+
+        # now we gotta append those property columns
+        # get the columns we care about
+        show_columns = [
+            # display name
+            Column.objects.filter(table_name="PropertyState", column_name=org.property_display_field, organization=org).first(),
+            # columns on the fpr
+            *list(fpr.display_columns.all()),
+            # building upgrade recommendation
+            Column.objects.filter(table_name="PropertyState", column_name="building_upgrade_recommendation", organization=org).first(),
```

Comment:
this line is giving me an error when I do not have a building_upgrade_recommendation column in my org